client/ts/README.md

```diff
@@ -1,51 +1,36 @@
+<a href="https://synnaxlabs.com/" style="display: block; text-align: center;">
+    <img src="../../x/media/static/logo/icon-white-on-black.png" width="20%"/>
+</a>
+
 # Synnax TypeScript Client Library
 
-The Synnax TypeScript client is a client library for interacting with a Synnax cluster.
-The client library can be used in both node and browser environments. The [Synnax documentation
-website](https://docs.synnaxlabs.com/typescript-client/get-started) contains more
-information about using the client.
+The Synnax TypeScript client library is used for interacting with a Synnax cluster. The
+client library can be used in both node and browser environments. The [Synnax
+documentation
+website](https://docs.synnaxlabs.com/reference/typescript-client/get-started) contains
+more information about using the TypeScript client.
 
 ## Installation
 
 If you want to install the client library, please install it via a package manager such
 as npm or yarn:
 
-```shell
-npm install @synnaxlabs/client
+```bash
+npm i @synnaxlabs/client
```

Comment:
I generally prefer using the whole `install` command, as it seems clearer to someone new to javascript.